test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -1293,15 +1293,16 @@ private static synchronized void showUI() {
 
 
     /**
-     * Forcibly pass the test.
-     * <p>The sample usage:
-     * <pre><code>
-     *      PrinterJob pj = PrinterJob.getPrinterJob();
-     *      if (pj == null || pj.getPrintService() == null) {
-     *          System.out.println(""Printer not configured or available.");
-     *          PassFailJFrame.forcePass();
-     *      }
-     * </code></pre>
+     * Forcibly pass the test. This method should be used in semi-automatic tests when
+     * the test determines that all the conditions for passing the test are met.
+     * <p>
+     * This method should not be used in cases where a resource is unavailable or a
+     * feature isn't supported, instead the test should
+     * throw jtreg.SkippedException.
+     *
+     * <p> A sample usage can be found in this test :
+     * <a href="https://github.com/openjdk/jdk/blob/master/test/jdk/java/awt/FileDialog/SaveFileNameOverrideTest.java#L84">SaveFileNameOverrideTest.java</a>
+     *
```

Comment:
```suggestion
     * Forcibly pass the test.
     * <p>
     * Use this method in semi-automatic tests when
     * the test determines that all the conditions for passing the test are met.
     * <p>
     * <strong>Do not use</strong> this method in cases where a resource is unavailable or a
     * feature isn't supported, throw {@code jtreg.SkippedException} instead.
     *
     * <p>A sample usage can be found in
     * <a href="https://github.com/openjdk/jdk/blob/7283c8b/test/jdk/java/awt/FileDialog/SaveFileNameOverrideTest.java#L84">{@code
     * SaveFileNameOverrideTest.java}</a>
```
This way looks better to me.

You want a permanent link; the test could change in the future.

Reply:
Yea, the line number might change, so I just removed the line number in the link.

Reply:
> Yea, the line number might change, so I just removed the line number in the link.

The link in my suggestion is tied to the particular revision, so the line number will never change.